src/components/cookie-consent/utils/analytics/matomo.js

```diff
@@ -0,0 +1,238 @@
+class Analytics {
+  constructor() {
+    this._matomoScriptId = 'vl-cookie-consent-matomo-script';
+    this._matomoPiwikScriptId = 'vl-cookie-consent-matomo-piwik-script';
+    this._matomoOntwikkelUrl = '//stats-ontwikkel.milieuinfo.be/';
+    this._matomoOefenUrl = '//stats-oefen.milieuinfo.be/';
+    this._matomoProdUrl = '//stats.milieuinfo.be/';
+  }
+
+  get scriptId() {
+    return this._matomoScriptId;
+  }
+
+  get id() {
+    let match = {
+      'stats-ontwikkel.milieuinfo.be': {
+        id: 1,
+        url: this._matomoOntwikkelUrl,
+      },
+      'ontwikkel.milieuinfo.be': {
+        id: 2,
+        url: this._matomoOntwikkelUrl,
+      },
+      'omgevingsloket2-ontwikkel.omgeving.vlaanderen.be': {
+        id: 3,
+        url: this._matomoOntwikkelUrl,
+      },
+      'bredero-ontwikkel.ruimteinfo.be': {
+        id: 5,
+        url: this._matomoOntwikkelUrl,
+      },
+      'bredero-bupo-ontwikkel.ruimteinfo.be': {
+        id: 6,
+        url: this._matomoOntwikkelUrl,
+      },
+      'bredero-xfr-ontwikkel.ruimteinfo.be': {
+        id: 7,
+        url: this._matomoOntwikkelUrl,
+      },
+      'ontwikkel.ruimtemonitor.be': {
+        id: 8,
+        url: this._matomoOntwikkelUrl,
+      },
+      'rupadviestoets-ontwikkel.milieuinfo.be': {
+        id: 9,
+        url: this._matomoOntwikkelUrl,
+      },
+      'zendantennes-ontwikkel.milieuinfo.be': {
+        id: 13,
+        url: this._matomoOntwikkelUrl,
+      },
+      'omgevingsloket-ontwikkel.omgeving.vlaanderen.be': {
+        id: 14,
+        url: this._matomoOntwikkelUrl,
+      },
+      'vsm-ontwikkel.milieuinfo.be': {
+        id: 16,
+        url: this._matomoOntwikkelUrl,
+      },
+      'mobiscore-ontwikkel.omgeving.vlaanderen.be': {
+        id: 22,
+        url: this._matomoOntwikkelUrl,
+      },
+      'erkenningencontactgegevens-ontwikkel.omgeving.vlaanderen.be': {
+        id: 24,
+        url: this._matomoOntwikkelUrl,
+      },
+      'loket-erkenningen-leefmilieu-ontwikkel.omgeving.vlaanderen.be': {
+        id: 30,
+        url: this._matomoOntwikkelUrl,
+      },
+      'bed-ontwikkel.omgeving.vlaanderen.be': {
+        id: 31,
+        url: this._matomoOntwikkelUrl,
+      },
```

Comment:
@LucasSledsens  Ik veronderstel dat we nog eens moeten kijken in de toekomst om deze lijst uit de component te halen? Lijkt me niet 100% ok dat dat in de component beheerd wordt niet?

Reply:
Inderdaad, zoals besproken wordt dit voorlopig gemigreerd zoals het was. In de toekomst bekijken hoe we dit beter kunnen aanpakken.